CONTRIBUTING.md

```diff
@@ -77,7 +77,7 @@ The steps are exactly the same for everyone involved in the project, including t
 1. If it's a new feature or a change of behavior, document it on the [akka-docs](https://github.com/akka/akka/tree/main/akka-docs). When the feature touches Scala and Java DSL, document both the Scala and Java APIs.
 1. Now it's finally time to [submit the pull request](https://help.github.com/articles/using-pull-requests)!
     - Please make sure to include a reference to the issue you're solving *in the comment* for the Pull Request, as this will cause the PR to be linked properly with the issue. Examples of good phrases for this are: "Resolves #1234" or "Refs #1234".
-1. If you have not already done so, you will be asked by our CLA bot to [sign the Lightbend CLA](https://www.lightbend.com/contribute/cla/akka) online. CLA stands for Contributor License Agreement and protects intellectual property disputes from harming the project.
+1. If you have not already done so, you will be asked by our CLA bot to [sign the Lightbend CLA](https://www.lightbend.com/contribute/cla) online. CLA stands for Contributor License Agreement and protects intellectual property disputes from harming the project.
```

Comment:
This is unfortunate. We changed this link in **all** Akka repositories based on the changes in https://github.com/akka/akka/pull/31566/files and https://github.com/akka/akka/pull/31562/files

Would it be worth adding a redirect or support both paths instead of changing all places again?

Reply:
We have a redirect already (see https://www.lightbend.com/contribute/cla/akka) - so this would only be cosmetics imo

Reply:
OK, if it is that much of a pain, let's just keep the redirect and be done with it